projects/dqn/experiments/base.py

```diff
@@ -0,0 +1,62 @@
+#  Numenta Platform for Intelligent Computing (NuPIC)
+#  Copyright (C) 2022, Numenta, Inc.  Unless you have an agreement
+#  with Numenta, Inc., for a separate license for this software code, the
+#  following terms and conditions apply:
+#
+#  This program is free software you can redistribute it and/or modify
+#  it under the terms of the GNU Affero Public License version 3 as
+#  published by the Free Software Foundation.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#  See the GNU Affero Public License for more details.
+#
+#  You should have received a copy of the GNU Affero Public License
+#  along with this program.  If not, see htt"://www.gnu.org/licenses.
+#
+#  http://numenta.org/licenses/
+#
+
+"""
+Base GSC Experiment configuration.
+"""
+
+import os
+
+import gym
+
+from nupic.research.frameworks.ray.trainables import DebugTrainable
+from src import DQN, EnvDataset, OffPolicyReinforcementLearningExperiment
```

Comment:
You should not import from `src`, it is confusing. This folder should not be used as the package name. If you are not creating an installable package then i would just remove the `src` folder to avoid confusion. 
See https://packaging.python.org/en/latest/tutorials/packaging-projects/

Reply:
changed to `main` as per discussion